NitroxClient/Unity/Helper/GameObjectHelper.cs

```diff
@@ -14,9 +14,9 @@ public static class GameObjectHelper
             return component;
         }
 
-        public static bool TryGetComponentInParent<T>(this GameObject go, out T component) where T : Component
+        public static bool TryGetComponentInParent<T>(this GameObject go, out T component, bool includeInactive = true) where T : Component
```

Comment:
Since `includeInactive = true` is not the default for Unity I would highlight this in a xml comment